decidim-core/spec/lib/icon_registry_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe IconRegistry do
+    subject { described_class.new }
+
+    let(:hash) { { "name" => "my Icon", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "action", "engine" => :core } }
+    let(:other_hash) { { "name" => "my Icon 1", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "other_category", "engine" => :core } }
+    let(:yet_another_hash) { { "name" => "my Icon 2", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "action", "engine" => :core } }
```

Comment:
If we want to add another example it'd be difficult to come up with another name. How about the good old `_N` suffix? 

```suggestion
    let(:icon_1) { { "name" => "my Icon 1", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "action", "engine" => :core } }
    let(:icon_2) { { "name" => "my Icon 2", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "other_category", "engine" => :core } }
    let(:icon_3) { { "name" => "my Icon 3", "icon" => "foo-bar", "resource" => "core", "description" => "some description", "category" => "action", "engine" => :core } }
```

Reply:
Our common friend would complain about `RSpec/IndexedLet`

Reply:
changed in b9005ca8d8